python/py-jsonpointer/Portfile

```diff
@@ -4,28 +4,30 @@ PortSystem          1.0
 PortGroup           python 1.0
 
 name                py-jsonpointer
-version             2.3
+version             2.4
 revision            0
 
 supported_archs     noarch
 platforms           {darwin any}
 license             BSD
 maintainers         nomaintainer
 
+python.pep517       yes
+
 description         Identify specific nodes in a JSON document (RFC 6901)
 long_description    {*}${description}
 
 homepage            https://github.com/stefankoegl/python-json-pointer
 
-checksums           sha256  97cba51526c829282218feb99dab1b1e6bdf8efd1c43dc9d57be093c0d69c99a \
-                    rmd160  76a846b62bab4adf17d1c1a845bc0baa9989fe3f \
-                    size    9295
+checksums           sha256  585cee82b70211fa9e6043b7bb89db6e1aa49524340dde8ad6b63206ea689d88 \
+                    rmd160  bf8e785646595a099dc92eaa9be4a4d55aed8445 \
+                    size    9254
 
-python.versions     37 38 39 310 311
+python.versions     37 38 39 310 311 312
 
 if {${name} ne ${subport}} {
-    depends_build-append \
-                    port:py${python.version}-setuptools
+    # depends_build-append \
+    #                port:py${python.version}-setuptools
```

Comment:
these lines should be removed, not commented out. Since these are all Python 3.7+ subports it will use `python.pep517 yes` with the default `setuptools` backend; so, indeed the `py-setuptools` dependency will be added automatically and doesn't need to be specified explicitly here anymore.